ansible/hosts.example.ini

```diff
@@ -0,0 +1,97 @@
+[all]
+cassandra01       ansible_host=X.X.X.X
+cassandra02       ansible_host=X.X.X.X
+cassandra03       ansible_host=X.X.X.X
+elasticsearch01   ansible_host=X.X.X.X
+elasticsearch02   ansible_host=X.X.X.X
+elasticsearch03   ansible_host=X.X.X.X
+minio01           ansible_host=X.X.X.X
+minio02           ansible_host=X.X.X.X
+minio03           ansible_host=X.X.X.X
+
+# * 'ansible_host' is the IP to ssh into
+# * 'ip' is the IP to bind to (if multiple network interfaces are in use)
+#   omit 'ip' if you only have one network interface
+# * etcd_member_name needs to be set on all hosts that run etcd (and must be different)
+kubenode01        ansible_host=X.X.X.X ip=Y.Y.Y.Y etcd_member_name=etcd1
+kubenode02        ansible_host=X.X.X.X ip=Y.Y.Y.Y etcd_member_name=etcd2
+kubenode03        ansible_host=X.X.X.X ip=Y.Y.Y.Y etcd_member_name=etcd3
+
+### databases ###
+
+[cassandra]
+cassandra00
+cassandra01
+cassandra02
+
+[cassandra_seed]
+cassandra00
+cassandra01
+
+[elasticsearch]
+elasticsearch00
+elasticsearch01
+elasticsearch02
+
+[elasticsearch_master:children]
+elasticsearch
+
+[minio]
+minio00
+minio01
+minio02
+
+### KUBERNETES ###
+
+[kube-master]
+kubenode00
+kubenode01
+kubenode02
+
+[kube-kubenode]
+kubenode00
+kubenode01
+kubenode02
+
+# must be an odd number of servers! (playbooks will fail otherwise)
+# See https://coreos.com/etcd/docs/latest/v2/admin_guide.html#optimal-cluster-size
+[etcd]
+kubenode00
+kubenode01
+kubenode02
+
+[k8s-cluster:children]
+kube-node
+kube-master
+
+[all:vars]
+### ANSIBLE section ###
+
+# ansible_ssh_private_key_file = path/to/key
+
+## default user to use to ssh
+# ansible_user = ...
+
+## use this if using python3 on the target machines
+ansible_python_interpreter = /usr/bin/python3
+
+## if you don't use ssh keys, enter the passwords to ssh and to become root:
+# ansible_ssh_pass = ...
+# ansible_become_pass = ...
+
+### KUBERNETES section (see kubespray documentation for details) ###
+
+bootstrap_os = ubuntu
+## set this to false if you have more than 3 nameservers
+# docker_dns_servers_strict = false
+
+[k8s-cluster:vars]
+# gotcha: use capital True or False (otherwise ansible parses as string)
+helm_enabled = True
+# 'flannel' is preferred on bare-metal setups, in case you wish to use `metallb`
+kube_network_plugin = flannel
+## these version should match the versions in the Makefile (to download the same versions of the API client binaries). Instead of upgrading these, you may wish to upgrade the version of kubespray. See sync_kubespray.yml
```

Comment:
Which versions in the `Makefile` does this refer to?

Reply:
They refer to versions specified in another PR: https://github.com/wireapp/wire-server-deploy/pull/71
Will fix the comment in that PR to avoid merge conflicts.